src/main.cpp

```diff
@@ -96,5 +124,9 @@ int main(int argc, char **argv)
 				 Q_ARG(QString, script));
 	}
 
-	return app.exec();
+
+	int ret = 0;
+	ret = app.exec();
+	return ret;
```

Comment:
I see no reason to create a temporary ret int. This can be left as "return app.exec()"